src/content/language/functions.md

```diff
@@ -620,3 +618,55 @@ is not `final`) an external setter.
 [conditional expression]: /language/operators#conditional-expressions
 [Flutter]: {{site.flutter}}
 [trailing commas]: /language/collections#lists
+
+## Getters and setters
+
+Every property access (top-level, static, or instance) is an invocation of a getter or a setter. A variable implicitly creates a getter and, if it's mutable, a setter. This is why when you access a property, you're actually calling a small function in the background. Reading a property calls a getter function, and writing one calls a setter function, even in cases where the property is declared a variable. 
+
+However, you can also declare a getter or a setter explicitly (using the keywords `get` or `set`). This allows a property's value to be computed when it's read or written.
+
+The purpose of using getters and setters is to create a clear separation between the client (the code that uses the property) and the provider (the class that defines it). The client asks for or sets a value without needing to know if that value is stored in a simple variable or calculated on the spot. This gives the provider the freedom to change how the property works. 
+
+For example, the value of the property might not be stored anywhere, it could be computed each time the getter is called. Another example is when a value is stored in a private variable, and public access is only allowed by calling a getter or a setter. The example below showcases this.
+
+```dart
+/// Defines a private variable secret with `_`.
+String _secret = 'Hello';
+
+/// The public top-level GETTER.
+/// It provides read access to the [_secret].
+String get secret {
+  print('Getter was used!');
+  return _secret.toUpperCase();
+}
+
+/// The public top-level SETTER.
+/// It provides write access to the [_secret].
+set secret(String newMessage) {
+  print('Setter was used! New secret: "$newMessage"');
+  if (newMessage.isNotEmpty) {
+    _secret = newMessage;
+  }
+}
+
+void main() {
+  // Reading the value calls the GETTER.
+  print('Current message: ${secret}');
+  // Output:
```

Comment:
I wonder if this might look better since it's meant for multi-line comments:

/*
Output:
Getter was used!
Current message: HELLO
*/

I would test and see what the preview looks like. If it looks good, I'd swap this and the other outputs, otherwise what you're using should be fine. I think this is a something we probably need to document in our documentation guide. I'll look into that.